webroot/js/components/chat/message.js

```diff
@@ -35,6 +35,7 @@ export default class Message extends Component {
               </div>
               <div
                 class="message-text text-gray-300 font-normal overflow-y-hidden"
+                title=${`Sent at ${formatTimestamp(timestamp)}`}
```

Comment:
Looks great @ahmadkarlam ! Could you declare this string up around line 19 with the other declaration, just for code style consistency?

Reply:
Done